frontend/components/LastUpdatedText/_styles.scss

```diff
@@ -1,5 +1,6 @@
 .component__last-updated-text {
   font-size: $xx-small;
+  font-weight: $regular;
```

Comment:
Updated globally here

Reply:
Thanks Rachel. Can you also please remove [these lines](https://github.com/fleetdm/fleet/blob/aef4bb579ed1dfbaddc648b5e4bfca2bef975011/frontend/pages/policies/ManagePoliciesPage/_styles.scss#L299-L301) (same fix, locally) with merging this global update